### PR TITLE
{bio}[foss/2018a] RELION v2.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -ur relion-2.1.orig/CMakeLists.txt relion-2.1/CMakeLists.txt
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch]: always claim ownership of a patch and write a single line what its function is

File: easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+--- relion-2.1.orig/CMakeLists.txt	2017-12-15 13:55:13.000000000 +0100
++++ relion-2.1/CMakeLists.txt	2018-06-01 11:20:17.908580307 +0200
+@@ -75,7 +75,7 @@
+     set(CUDARCH "-arch=sm_35")
+ else(NOT DEFINED CUDA_ARCH)
+     message(STATUS "Using provided CUDA_ARCH=${CUDA_ARCH}")
+-    set(CUDARCH "-arch=sm_${CUDA_ARCH}")
++    set(CUDARCH ${CUDA_ARCH})
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch]: this is a bit weird? the else condition is `NOT DEFINED CUDA_ARCH` and then you just use it?

Comment (akesandgren) [reply easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch]: Nah, the (NO DEFINED...) is a comment referring to the actual if-stmt

Comment (smoors) [reply easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch]: yes. I agree it's confusing if you see this for the first time

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic170 - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz, Python 2.7.5
See https://gist.github.com/23267bab396b317bd073b587000d788a for a full test report.

**Comment by boegel** [comment]
@wpoely86 good to go?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2613.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/52aec569282fd610015d7868f54b59df for a full test report.

**Pull request merged**